includes/fields/class-password.php

```diff
@@ -62,6 +62,6 @@ protected function sanitize() {}
 	 * @return string
 	 */
 	public function render() {
-		return '<input type="' . esc_attr( $this->display_type ) . '" name="' . esc_attr( $this->name ) . '" ' . hp\html_attributes( $this->attributes ) . '>';
+		return '<input type="' . esc_attr( $this->display_type ) . '" name="' . esc_attr( $this->name ) . '" ' . hp\html_attributes( $this->attributes ) . '><div id="hp-password-strength-result"></div>';
```

Comment:
Make sure that this ID is required by the script you loaded, otherwise it's better to use data-component=password and keep all the front-end logic like inserting this DIV in JS.